cartoframes/data/enrichment/points_enrichment.py

```diff
@@ -95,20 +96,19 @@ def _prepare_sql(enrichment_id, filters_processed, table_to_geotable, table_to_v
         sql = '''
             SELECT data_table.{enrichment_id},
                 {variables},
-                ST_Area(enrichment_geo_table.geom) AS {variables_underscored}_area,
-                NULL AS {variables_underscored}_population
+                ST_Area(enrichment_geo_table.geom) AS {enrichment_table}_area
             FROM `{project}.{dataset}.{enrichment_table}` enrichment_table
             JOIN `{project}.{dataset}.{enrichment_geo_table}` enrichment_geo_table
             ON enrichment_table.geoid = enrichment_geo_table.geoid
             JOIN `{working_project}.{user_dataset}.{data_table}` data_table
             ON ST_Within(data_table.{data_geom_column}, enrichment_geo_table.geom)
             {filters};
-        '''.format(enrichment_id=enrichment_id, variables=', '.join(variables),
-                   variables_underscored='_'.join(variables), enrichment_table=table,
-                   enrichment_geo_table=table_to_geotable[table], user_dataset=user_dataset,
-                   working_project=working_project, data_table=data_table,
+        '''.format(enrichment_id=enrichment_id, variables_underscored='_'.join(variables),
```

Comment:
`variables_underscored` is not being used

Reply:
Removed unused variable from format method!